habitat_baselines/evaluate_ppo.py

```diff
@@ -5,20 +5,59 @@
 # LICENSE file in the root directory of this source tree.
 
 import argparse
-
+import cv2
+import numpy as np
+import os
 import torch
-
+import time
 import habitat
+from habitat import logger
+from torch.utils import tensorboard
+from habitat.utils.visualizations.utils import images_to_video
 from config.default import get_config as cfg_baseline
 from habitat.config.default import get_config
 from rl.ppo import PPO, Policy
 from rl.ppo.utils import batch_obs
 from train_ppo import make_env_fn
+from habitat.utils.visualizations import maps
+
+
+def poll_checkpoint_folder(checkpoint_folder, previous_ckpt_ind):
+    assert os.path.isdir(checkpoint_folder), "invalid checkpoint folder path"
```

Comment:
I don't have background here, but when the checkpoint is not there in case of starting, should it not silently ignore and return?

Reply:
In main() it constantly checks for new checkpoints. If not found it will sleep for 2 sec and check again.